tasks/xpack/security/elasticsearch-security-file.yml

```diff
@@ -72,7 +72,7 @@
   become: yes
   command: >
     {{es_home}}/bin/{{es_xpack_users_command}} passwd {{ item }} -p {{es_users.file[item].password}}
-  with_items: "{{ es_users.file.keys() | default([]) }}"
```

Comment:
Can the `default([])` be removed here? Does list automatically return an empty list if the variable is undefined? 

It fails for me on https://ansible.sivel.net/test/ but maybe it doesn't handle empty variables properly.

Reply:
With the `when: manage_file_users` and `set_fact: manage_file_users=es_users is defined and es_users.file is defined and es_users.file.keys() | list | length > 0` at the top of the file, if the variable is undefined we will skip this task so I think we can avoid having an additional `default` filter here, WDYT.

Reply:
Makes sense to me!